src/SDKs/CognitiveServices/dataPlane/Language/LUIS/Programmatic.sln

```diff
@@ -1,50 +0,0 @@
-Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
```

Comment:
I see `Programmatic.sln` is being replaced by `Authoring.sln` but the csproj names are still the same, please keep consistency, same goes for the package name
On a related note, is your intention to merely rename the directories or the name of the package too